dhis-2/dhis-web/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/JobConfigurationControllerTest.java

```diff
@@ -52,6 +52,10 @@
  */
 public class JobConfigurationControllerTest extends DhisControllerConvenienceTest
 {
+    private static final String PROGRAM1 = "Information Campaign";
```

Comment:
Use UIDs in tests to make tests more realistic.

Reply:
done